tests/components/thethingsnetwork/test_config_flow.py

```diff
@@ -0,0 +1,121 @@
+"""Define tests for the The Things Network onfig flows."""
```

Comment:
I'm missing the test for duplicate entry, that check if we abort if we input the same

Reply:
Added test_duplicate_entry